conf/nginx/nginx.conf

```diff
@@ -10,87 +10,60 @@ upstream gnupstream {
 	server gunicorn:8060;
 }
 
-server {
-	listen 80;
-	server_name localhost;
-
-	location /static/ {
-	    alias /static/;
-	}
-
-	location /his {
-		proxy_pass http://gnupstream;
-	}
-
-	location /geoserver {
-		proxy_pass http://gsupstream;
-	}
-
-	location /flower {
-		proxy_pass http://flupstream;
-	}
-
-    proxy_connect_timeout       3605;
-    proxy_send_timeout          3605;
-    proxy_read_timeout          3605;
-    send_timeout                3605;
-    keepalive_timeout           3605;
-
-}
-
 # server {
 #     listen 80;
 #     server_name localhost;
-
+#
 #     return 301 https://$server_name$request_uri;
 # }
 
-# server {
-#     listen 443 ssl;
-#     server_name localhost;
+ server {
+     #listen 443 ssl;
+     listen 80;
+     server_name localhost
```

Comment:
these nginx conf changes are just because we are deploying in cloud...minimizing the diffs that we need in production, which will only listen 80 (LB has the cert)